Purchasing.Mvc/Attributes/AuthenticateListView.cs

```diff
@@ -1,42 +1,52 @@
 ﻿using System;
+using System.Threading.Tasks;
 using System.Text;
-using System.Web.Configuration;
-using System.Web.Mvc;
+using System.Configuration;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Configuration;
+using UCDArch.Core;
 
 namespace Purchasing.Mvc.Attributes
 {
     public class AuthenticateListView : ActionFilterAttribute
     {
+        public const string BLOCK_401_TO_302_REDIRECT = "Block401To302Redirect";
```

Comment:
We can't really test this unless we point sympa to this instance, but we don't really use this anyway....